core/templates/dev/head/components/common-layout-directives/common-elements/lazy-loading.directive.html

```diff
@@ -0,0 +1,26 @@
+<div class="oppia-lazy-loading-container">
+  <div class="oppia-mover"></div>
```

Comment:
What's a "mover"? I don't understand the term... perhaps choose a more understandable name? (Unless this is a standard UI component, with that name.)